src/borg/fuse.py

```diff
@@ -242,16 +242,15 @@ def _create_filesystem(self):
             self.process_archive(self.args.location.archive)
         else:
             self.versions_index = FuseVersionsIndex()
-            archive_names = (x.name for x in self.manifest.archives.list_considering(self.args))
-            for archive_name in archive_names:
+            for archive in self.manifest.archives.list_considering(self.args):
                 if self.versions:
                     # process archives immediately
-                    self.process_archive(archive_name)
+                    self.process_archive(archive.name)
                 else:
                     # lazily load archives, create archive placeholder inode
-                    archive_inode = self._create_dir(parent=1)
-                    self.contents[1][os.fsencode(archive_name)] = archive_inode
-                    self.pending_archives[archive_inode] = archive_name
+                    archive_inode = self._create_dir(parent=1, mtime=int(archive.ts.timestamp() * 1e9))
```

Comment:
I see the `have_fuse_xtime_ns` code, but doesn't that run for the `EntryAttributes` object and convert it there? Why do I need to convert it if it's not necessary for [`self.default_dir`](https://github.com/borgbackup/borg/blob/master/src/borg/fuse.py#L232) here?